wasm_src/ast_wrapper/ast_wrapper.cc

```diff
@@ -80,6 +80,135 @@ EMSCRIPTEN_KEEPALIVE AstFrameSet* initFrame(const char* header)
     return wcsinfo;
 }
 
+EMSCRIPTEN_KEEPALIVE AstSpecFrame* initSpectralFrame(const char* header, const char *system, const char *unit)
+{
+     if (!header || !system || !unit)
+    {
+        cout << "Missing header argument." << endl;
+        return nullptr;
+    }
+
+    AstFitsChan* fitschan = nullptr;
+    AstFrameSet* frame = nullptr;
+    int status = 0;
+    if (frame)
+    {
+        astEnd;
+    }
+	astClearStatus;
+    astBegin;
+
+    fitschan = astFitsChan(NULL, NULL, "");
```

Comment:
should use `nullptr` instead (please update my `NULL` values at the same time :)